sync.py

```diff
@@ -99,6 +103,40 @@ def check_unsynced_modules(tagged_builds, modules_to_track):
 
     return unsynced_builds
 
+def check_cve_builds(tagged_builds):
```

Comment:
Shall we rename it to `filter_embargoed_builds()` ?

Reply:
Sure, I'd named it in a similar style to check_unsynced_builds().